app/src/main/res/values/strings.xml

```diff
@@ -571,39 +571,37 @@
-        cases your GitHub account Oauth token will do the trick.</p>
+        <h5>• I have signed into my Enterprise account. Why can\'t I interact with anything other than my Enterprise GitHub?</h5>
+        <p>Well, logically, you can\'t access anything else other than your Enterprise. FastHub tries to require as much permissions as possible, but can\'t do anything more about it
+        in most cases, since your credentials are not on GitHub server. But in <b>some</b> cases your GitHub account OAuth token will do the trick.</p>
 
-        <h5>• Why am I having problems editing Issues/PRs?</h5>
-        <p>If you are editing a public Org repo, then please contact your Org to grant access to FastHub or use Access Token to login!.</p>
+        <h5>• Why am I having problems editing Issues/Pull Requests (PRs)?</h5>
+        <p>If you are unable to edit Issues/PRs in a Public Organization's repository, please contact your Organization to grant access to FastHub. Alternatively you can sign in with an Access Token.</p>
 
-        <h5>• I\'m having this issue or I want this &amp; that!!</h5>
-        <p>Head to https://github.com/k0shk0sh/FastHub/issues/new and create new issue for bugs or feature requests, I really do encourage you to
-        search before opening a ticket. Any duplicate request will result in it being closed immediately.</p>
+        <h5>• I\'m having problems! / I want to suggest you some things!</h5>
+        <p>Head to https://github.com/k0shk0sh/FastHub/issues/new and create a new issue for any bugs and feature requests. I really do encourage you to
+        search for similar issues before opening a new one. Any duplicated ticket will be labeled as duplicated and closed immediately.</p>
 
-        <h5>• How do I get PROMO CODE?</h5>
-        <p>If you are a student, you\'ll have to provide me via Email that you are student, you will need below documents:</p>
+        <h5>• How can I get a PROMO CODE?</h5>
+        <p>If you are a student, you\'ll have to prove me that you are student. You will need to send me via Email some proofs listed below:</p>
         <ul>
-        <li>Your university identity card & your identity card (that shows your name & your face to compare it!)</li>
-        <li>Your university start & end date</li>
-        <li>Rate FastHub in the Play Store</li>
+        <li>Your student card and your identity card/passport (that shows your name and face to compare them!);</li>
+        <li>Your education start and graduation dates.</li>
         </ul>
-        <p>If you aren\'t a student and you can\'t afford to pay for PRO, you\'ll need:</p>
+        <p>If you aren\'t a student and you can\'t afford purchasing PRO, you\'ll need to:</p>
         <ul>
-        <li>Write an article about FastHub in social media such as (Medium)</li>
-        <li>Rate FastHub in the Play Store</li>
+        <li>Write an article about FastHub in any social media such as (Medium);</li>
```

Comment:
the `(` should start before such as

Reply:
Don't agree. This line could be extended with additional social media. 
E.g. `Write an article about FastHub in any social media such as (Medium), (Twitter), (Facebook), (VK), etc.`

Reply:
(such as Medium, Twitter, Facebook, VK), 😄

Reply:
But the links are different for the websites.

Reply:
so what? such as belong inside the parentheses

Reply:
`[Medium](url), [Twitter](url), [Facebook](url), [VK](url), etc.`

Reply:
the words ok understand change it to links....

Reply:
I don't get the meaning of parentheses here. This is how I understand them.

Reply:
OK you push your update and i will fix if i have an issue

Reply:
Let me explain. 

Parentheses are user when you want to add something that is not in context. The sentence should make sense without it. The use is to explain what mean with something you said.

So in our case if you take out the parentheses (including its word), the sentence will not make sense.

`Write an article about FastHub in any social media such as`

> So in our case if you take out the parentheses (including its word), the sentence will not make sense.
In the above case i used parentheses lets take it out and my sentence will make sense.
>So in our case if you take out the parentheses, the sentence will not make sense.